faucet/faucet.py

```diff
@@ -101,6 +102,7 @@ class Faucet(RyuAppBase):
     metrics = None
     notifier = None
     valves_manager = None
+    event_socket_heartbeat_time = 0
```

Comment:
typically better to pospend with units like "ms" or "sec" -- because just looking at this code I can't tell which one it is (and then _time is unnecessary)

Ideally this would be on the ENV variable too, but FAUCET_EVENT_SOCKET_HEARTBEAT_MS is a bit long even for me :-P

Reply:
added seconds as comment.